enaml/src/dynamicscope.cpp

```diff
@@ -634,6 +634,10 @@ DynamicScope_getitem( DynamicScope* self, PyObject* key )
     if( strcmp( (char *)Py23Str_AS_STRING( key ), "__scope__" ) == 0 )
         return newref( pyobject_cast( self ) );
 
+    // _[tracer] magic
+    if( strcmp( (char *)Py23Str_AS_STRING( key ), "_[tracer]" ) == 0 )
```

Comment:
It's possible for `self->tracer` to be `null` so I think you want:

`if( self->tracer && strcmp(...) == 0)`

Reply:
Fixed